README.md

```diff
@@ -19,9 +19,7 @@ Other times, you may find that a modifier is not the right fit for that logic at
 in which case it's worth revisiting the design to find a better pattern.
 
 Either way, we recommend using these modifiers with caution. They are very useful for
-quickly bridging the gap between classic components and Glimmer components, but they
-are still generally an anti-pattern. We recommend considering a custom modifier in
```

Comment:
just 2 cents here.
IMO `{{did-update}}` is really an anti-pattern.

As for did-insert/will-destroy, my personal opinion is to create custom local modifier when app running Ember 3.25+ otherwise keep using `did-insert`/`will-destroy`